charts/airbyte-webapp/values.yaml

```diff
@@ -178,5 +173,8 @@ extraVolumeMounts: []
 extraVolumes: []
 
 extraContainers: []
+<<<<<<< HEAD
 
 secrets: {}
+=======
+>>>>>>> 8a2d0ce928 (remove demo mode from UI)
```

Comment:
Unresolved conflict, whoops

Reply:
Good catch! Looks like I used the vs code "accept incoming change" button incorrectly somehow. Pushed the fix!